cardano-launcher/app/Main.hs

```diff
@@ -32,40 +35,39 @@ import           Data.X509.Extra (failIfReasons, genRSA256KeyPair,
 -- Main
 --------------------------------------------------------------------------------
 
--- | Yes, this is something we probably need to replace with actual loggging.
--- If we want. Otherwise, we can print out to console out just for launcher configs.
-externalDependencies :: ExternalDependencies
-externalDependencies = ExternalDependencies
-    { logInfo            = putTextLn
-    , logError           = putTextLn
-    , logNotice          = putTextLn
-    }
-
 -- | Main function.
 main :: IO ()
 main = do
-
-    let launcherConfig :: LauncherConfig
-        launcherConfig = LauncherConfig
-            { lcfgFilePath    = "./configuration/cert-configuration.yaml"
-            , lcfgKey         = "dev"
-            , lcfgSystemStart = Nothing
-            , lcfgSeed        = Nothing
-            }
+    -- Todo: make it so that you can specify the path via CLI
+    launcherOptions <- either
+        (\err -> throwM $ LauncherOptionsError (show err))
+        return
+        =<< getLauncherOption "./cardano-launcher/configuration/launcher/launcher-config.demo.yaml"
```

Comment:
id like to see some CLI args working before this gets merged
and it should default to `launcher-config.yaml` in the same dir as the executable, as it did before:
https://github.com/input-output-hk/cardano-sl/blob/develop/tools/src/launcher/Main.hs#L188-L197

Reply:
I have this ready on the next PR.